assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -439,3 +452,23 @@ bool EntityTreeSendThread::traverseTreeAndBuildNextPacketPayload(EncodeBitstream
 #endif // SEND_SORTED_ENTITIES
 }
 
+void EntityTreeSendThread::editingEntityPointer(const EntityItemPointer entity) {
+    if (entity) {
+        if (_entitiesInQueue.find(entity.get()) == _entitiesInQueue.end() && _knownState.find(entity.get()) != _knownState.end()) {
+            bool success = false;
+            AACube cube = entity->getQueryAACube(success);
+            if (success) {
+                float priority = _conicalView.computePriority(cube);
+                _sendQueue.push(PrioritizedEntity(entity, priority, true));
+                _entitiesInQueue.insert(entity.get());
+            } else {
+                _sendQueue.push(PrioritizedEntity(entity, PrioritizedEntity::WHEN_IN_DOUBT_PRIORITY, true));
+                _entitiesInQueue.insert(entity.get());
+            }
+        }
+    }
+}
+
+void EntityTreeSendThread::deletingEntityPointer(EntityItem* entity) {
+    _knownState.erase(entity);
+}
```

Comment:
This is the only place where something might be removed from `_knownState`, except where we clear `_knownState` before the first traversal.